app/react/Settings/components/CollectionSettings.js

```diff
@@ -322,7 +322,21 @@ export class CollectionSettings extends Component {
               <label className="form-group-label" htmlFor="collectionContactEmail">
                 <Translate>Contact email</Translate>
               </label>
-              <Control.text model=".contactEmail" className="form-control" />
+              <Control.text
+                id="collectionContactEmail"
+                model=".contactEmail"
+                className="form-control"
+              />
+            </div>
+            <div className="form-group">
+              <label className="form-group-label" htmlFor="collectionSenderEmail">
+                <Translate>Sender email</Translate>
+              </label>
+              <Control.text
+                id="collectionSenderEmail"
+                model=".senderEmail"
+                className="form-control"
+              />
```

Comment:
Probably this option requires some explanation here, or in the wiki

Reply:
I have added an explanation, here is how it looks like
![image](https://user-images.githubusercontent.com/8601175/98541719-ed859b00-22a0-11eb-8b42-2c640e692daf.png)